libraries/Adapters/Microsoft.Bot.Builder.Adapters.Twilio/TwilioAdapter.cs

```diff
@@ -0,0 +1,248 @@
+            {
+                throw new Exception("TwilioNumber is a required part of the configuration.");
+            }
+
+            if (string.IsNullOrWhiteSpace(options.AccountSid))
+            {
+                throw new Exception("AccountSid is a required part of the configuration.");
+            }
+
+            if (string.IsNullOrWhiteSpace(options.AuthToken))
+            {
+                throw new Exception("AuthToken is a required part of the configuration.");
+            }
+
+            _options = options;
+
+            TwilioClient.Init(_options.AccountSid, _options.AuthToken);
+        }
+
+        /// <summary>
+        /// Standard BotBuilder adapter method to send a message from the bot to the messaging API.
+        /// </summary>
+        /// <param name="turnContext">A TurnContext representing the current incoming message and environment.</param>
+        /// <param name="activities">An array of outgoing activities to be sent back to the messaging API.</param>
+        /// <param name="cancellationToken">A cancellation token for the task.</param>
+        /// <returns>A resource response.</returns>
+        public override async Task<ResourceResponse[]> SendActivitiesAsync(ITurnContext turnContext, Activity[] activities, CancellationToken cancellationToken)
+        {
+            var responses = new List<ResourceResponse>();
+            foreach (var activity in activities)
+            {
+                if (activity.Type == ActivityTypes.Message)
+                {
+                    var messageOptions = ActivityToTwilio(activity);
+
+                    var res = await MessageResource.CreateAsync(messageOptions).ConfigureAwait(false);
+
+                    var response = new ResourceResponse()
+                    {
+                        Id = res.Sid,
+                    };
+
+                    responses.Add(response);
+                }
+                else
+                {
+                    throw new Exception("Unknown message type");
+                }
+            }
+
+            return responses.ToArray();
+        }
+
+        /// <summary>
+        /// Accept an incoming webhook request and convert it into a TurnContext which can be processed by the bot's logic.
+        /// </summary>
+        /// <param name="request">A request object from Restify or Express.</param>
+        /// <param name="response">A response object from Restify or Express.</param>
+        /// <param name="bot">A bot with logic function in the form `async(context) => { ... }`.</param>
+        /// <param name="cancellationToken">A cancellation token for the task.</param>
+        /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
+        public async Task ProcessAsync(HttpRequest request, HttpResponse response, IBot bot, CancellationToken cancellationToken = default)
+        {
+            response.StatusCode = 200;
```

Comment:
In general, use response.StatusCode = (int)HttpStatusCode.OK; rather than hardcoding 200 (just in case the status codes change in 10 years 😊)

Reply:
We'll apply this on the refactor. Thanks!